### PR TITLE
media-libs/libextractor: version bump to 1.6

### PR DESCRIPTION
**Pull request by gitbugged**
New version released 15-Oct-2017 19:39.

Signed-off-by: Bob Brooks <gitbugged@cool.fr.nf>

### CODE REVIEW

File: media-libs/libextractor/libextractor-1.6.ebuild [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+	e_ac_cv header_zlib_h=$(usex zlib)
+	e_ac_cv lib_mp4v2_MP4ReadProvider=$(usex mp4)
+	e_ac_cv lib_smf_smf_load_from_memory=$(usex midi)
+
+	# gstreamer support is for 1.0, no 0.10 support
+	econf \
+		--disable-static \
+		--enable-experimental \
+		--enable-glib \
+		--without-gstreamer \
```
Comment (a17r) [inline media-libs/libextractor/libextractor-1.6.ebuild]: Reading above comment, which seems is left over from a time where 1.0 was not in tree, it seems we should provide USE=gstreamer again, could you do that please?

Comment (gitbugged) [reply media-libs/libextractor/libextractor-1.6.ebuild]: @a17r Is the proper solution to just add "$(use_enable gstreamer gstreamer-1-0)" to the bottom of the list, or is there more needed? Was unable to find an example during a search.

Comment (a17r) [reply media-libs/libextractor/libextractor-1.6.ebuild]: `configure.ac` contains `AC_ARG_WITH([gstreamer]` so we can do `$(use_with gstreamer)`, but I've already prepared that in a subsequent commit.

**Comment by a17r** [comment]
@gitbugged, are you interested in proxy-maintaining this package? I'll be adding some commits post-merge but otherwise I personally have no need for libextractor.

**Comment by a17r** [comment]
Relevant commits by me after merge you should check out:
5d8121bb17a428dadb74497a612e4e0f9ffd457f: Re-add USE=gstreamer
26b21b170d77a21ffbbd755947bf7ab2bb7944e7: Add USE=apparmor (as an example using `e_ac_cv`)
f144d034d3f2b2f9551dfb341d1b6dcdb2a4aabf: Fix USE=tidy
6859a8b699efc9cd61a9eede139220391494d14b: Patch CVE-2017-17440

**Pull request closed without merging**